packages/mui-styles/src/ServerStyleSheets/ServerStyleSheets.d.ts

```diff
@@ -5,7 +5,7 @@ declare class ServerStyleSheets {
   constructor(options?: object);
   collect(children: React.ReactNode, options?: object): React.ReactElement<StylesProviderProps>;
   toString(): string;
-  getStyleElement(props?: object): React.ReactElement<any>;
+  getStyleElement(props?: object): React.ReactElement<unknown>;
```

Comment:
Let's not touch `mui-styles` as it's not being updated. Same for the other `mui-styles` file.

Reply:
reverted here https://github.com/mui/material-ui/pull/43402/commits/b58331026b9e90d8f3a3b669990e09bbc11e435d